### PR TITLE
feat: support multiple organizations

### PR DESCRIPTION
**Pull request by zhengxs2018**

### CODE REVIEW

**Comment by transitive-bullshit** [comment]
Great change – thanks! 🙏

File: src/types.ts [hunk 1]
```diff
@@ -10,6 +10,8 @@ export type ChatGPTAPIOptions = {
   /** @defaultValue `'https://api.openai.com'` **/
   apiBaseUrl?: string
 
+  apiOrg: string
```
Comment (alxmiron) [inline src/types.ts:13]: @zhengxs2018 @transitive-bullshit Why it's not optional? Not it requires me to pass it to the constructor

Comment (transitive-bullshit) [reply src/types.ts:13]: ahhh will fix this now; def a bug

Comment (transitive-bullshit) [reply src/types.ts:13]: should be fixed now @alxmiron https://github.com/transitive-bullshit/chatgpt-api/releases/tag/v5.2.2

**Comment by transitive-bullshit** [comment]
`apiOrg` is now optional in https://github.com/transitive-bullshit/chatgpt-api/releases/tag/v5.2.2

**Comment by zhengxs2018** [comment]
> https://github.com/transitive-bullshit/chatgpt-api/releases/tag/v5.2.2中是可选的

nice~

File: src/types.ts (hunk 1 above)
Comment (zhengxs2018) [reply src/types.ts:13]: @alxmiron  Sorry, It should be optional.

**Pull request merged**